src/components/SearchInput.vue

```diff
@@ -155,11 +155,13 @@ export default {
     value: {
       type: String,
       required: false,
+      default: undefined,
```

Comment:
what is this needed for?

Reply:
eslint gave me this

```
/home/circleci/vue-instantsearch/src/components/SearchInput.vue
  155:5  error  Prop 'value' requires default value to be set       vue/require-default-prop
  159:5  error  Prop 'modelValue' requires default value to be set  vue/require-default-prop
```

I'm not sure why it didn't alert earlier.